src/app/components/form/form.component.ts

```diff
@@ -60,6 +64,7 @@ export class FormComponent implements OnInit {
 
   onGameChanged(event: Event): void {
     this.game = Number((event.target as HTMLInputElement).value)
+    this.showTeams()
```

Comment:
Maybe we should add this for onEventChanged also?

Reply:
i didnt understood what you mean here